app/jobs/generate_ptiff_from_mets_job.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class GeneratePtiffFromMetsJob < ApplicationJob
+  queue_as :ptiff
+
+  def default_priority
+    10
+  end
+
+  def perform(child_object, current_batch_process, current_batch_connection = child_object.parent_object.current_batch_connection)
+    child_object.parent_object.current_batch_process = current_batch_process
+    child_object.parent_object.current_batch_connection = current_batch_connection
```

Comment:
there is a lot of duplication here. I want to respect your comment about drying it up after this gets in, but I wonder if a quick toweling off of the jobs now might be worth while. If the broader structural change of making "from_mets" a parent_object property happens (see below) then this could simply be collapsed in to the original job again.  We might want to separate concerns from there, or split out a callback, but that would seem more like refactoring to me.